config/database.yml

```diff
@@ -80,6 +80,8 @@ test:
 #
 production:
   <<: *default
-  database: helloworld_production
-  username: helloworld
-  password: <%= ENV['HELLOWORLD_DATABASE_PASSWORD'] %>
+  database: <%= ENV['APP_DB_DATABASE'] %> # helloworld_production
```

Comment:
@AdrianTeh 
I'd suggest using `ENV.fetch('APP_DB_DATABASE')` for these calls so the container will crash if the environment values do not exist.